tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
```

Comment:
Doesn't this have a potential to break stuff? https://www.typescriptlang.org/tsconfig/#baseUrl

Reply:
I don't think so (but I could be wrong). Reanimated also has it in top-level `tsconfig`. Moreover, all nested `tsconfigs` extend this one, so I assume they will treat it as the `baseUrl` points to their directory.